src/lib/button-toggle/button-toggle.scss

```diff
@@ -15,7 +15,6 @@ $mat-button-toggle-legacy-border-radius: 2px !default;
   position: relative;
   display: inline-flex;
   flex-direction: row;
-  cursor: pointer;
   white-space: nowrap;
   overflow: hidden;
```

Comment:
Note: I've kept `white-space` and `overflow` since they don't cause any issues yet, and otherwise we could potentially break apps.